### DISCUSSION TITLE
能android上运行么？

### DISCUSSION THREAD

**Discussion by ChenyangS** [SeaHOH/GotoX#107] [问答/咨询] [answered]
因为在Win10上有莫名奇妙的bug，想在Android模拟器上搞，找了个Qpython3 貌似不行

![2019-03-04_202740](https://user-images.githubusercontent.com/35072138/53733233-5a854280-3ebb-11e9-9436-09f91321c2cc.png)

各位有什么方案么？

**Comment by SeaHOH** [comment]
估计很麻烦，要搞定依赖。可以去请教 XndroidDev。

话说，上面这个哪里是语法错误？完全看不出来。

**Comment by majiayeah** [comment]
可以在Termux中运行。
Termux中安装python，再pip安装所有依赖库。

**Comment by SeaHOH** [comment, marked as answer]
> 可以在Termux中运行。
> Termux中安装python，再pip安装所有依赖库。

#114
